sdk/eventhub/eventhubs-checkpointstore-table/test/tables-checkpointstore.spec.ts

```diff
@@ -0,0 +1,556 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import chai from "chai";
+import * as dotenv from "dotenv";
+const should = chai.should();
+import { TableCheckpointStore } from "../src";
+import debugModule from "debug";
+const debug = debugModule("azure:event-hubs:partitionPump");
+import { Checkpoint, PartitionOwnership } from "@azure/event-hubs";
+import { EnvVarKeys, getEnvVars } from "./utils/testUtils";
+import { TableServiceClient, AzureNamedKeyCredential, TableClient } from "@azure/data-tables";
+import { CheckpointEntity, PartitionOwnershipEntity } from "../src/tableCheckpointStore";
+
+const env = getEnvVars();
+dotenv.config();
+
+/* test to show that test framework is set up well */
+describe("TableCheckpointStore", () => {
+  it("is exported from the package", () => {
+    should.exist(TableCheckpointStore, "Expected TableCheckpointStore to be exported.");
+  });
+});
+
+const service = {
+  storageConnectionString: env[EnvVarKeys.STORAGE_CONNECTION_STRING],
+  storageAccountName: env[EnvVarKeys.STORAGE_ACCOUNT_NAME],
+  storageAccountKey: env[EnvVarKeys.STORAGE_ACCOUNT_KEY],
+  tableName: env[EnvVarKeys.TABLENAME]
+};
+
+const credential = new AzureNamedKeyCredential(
+  service.storageAccountName,
+  service.storageAccountKey
+);
+const serviceClient = new TableServiceClient(
+  `https://${service.storageAccountName}.table.core.windows.net`,
+  credential
+);
+
+describe("TableCheckpointStore", function(): void {
+  let client: TableClient;
+  let table_name: string;
+
+  describe("Runs tests on table with no entities", function() {
+    table_name = `table${new Date().getTime()}`;
```

Comment:
You might still want to add a random number as well. When the CI runs these tests, they may start at the same time (as unlikely as it is).

Reply:
ok